libs/payment/api/src/lib/payment-provider/bexio/bexio-payment-provider.ts

```diff
@@ -232,7 +249,7 @@ export class BexioPaymentProvider extends BasePaymentProvider {
 
       return {
         intentID: newInvoice.id.toString(),
-        intentSecret: '',
+        intentSecret: successURL,
```

Comment:
Let's have a call about that. It's faster that way, just call me when you got time.

Reply:
Can we have a summary here? I guess it's for the redirect on the frontend?

Reply:
Sure. Intent is where the frontend is redirected to do the payment (for payrexx and stripe checkout). In case of Bexio the invoice is already created, so we just redirect the UI to the success url because in case of Bexio there is no extra action required on a bexio page. It enables  bexio integration in the fronted without any significant changes.